arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -83,22 +108,40 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           }
         ],        
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. Returns a list of all of the Media Services accounts in a resource group.",
             "schema": {
               "$ref": "#/definitions/MediaServiceCollection"
+            },
+            "examples": {
+              "id": "/subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/Microsoft.Media/mediaServices/{name}",
+              "name": "{name}",

```

Comment:
Please dont use templates. Just provide the right data. This will help the tooling in validating that the sample is correct as per the schema.